### DISCUSSION TITLE
Final project pitch

### DISCUSSION THREAD

**Discussion by biavillasboas** [archive-GPGN-268-SP2024/GPGN268-CORE#4] [General]
Hi folks, 
what topics/problems are you interested in exploring on your final projects? Here are some areas that you could focus on:

- 🌎 **Climate Geophysics:** oceanography, surface water distribution, glaciology, climate change, permafrost ...
- ⚡️**Energy Geophysics:** hydrocarbon resources, geothermal resources, Carbon capture ...
- 💎 **Minerals Geophysics:** sustainable mining, deep-sea mining, critical mineral resources...
- 🌋 **Hazards Geophysics:** extreme weather (wildfires, flooding, drought), earthquakes, volcanos, landslides...
- 🪐 **Space Geophysics:** Structure of comets and asteroids, Mars characterization, lunar resources, solar storms...

Pitch your project ideas below!

**Comment by lexiherr** [comment]
climate or space geophysics!

**Reply by lexiherr** [reply]
I would love to do something about tracking environmental changes through sea ice data

**Comment by anna-nichols** [comment]
Space geophysics or hazard geophysics!

**Comment by LM2025** [comment]
Hazard or Climate Geophysics sound particularly interesting! I think it would be nice to explore geophysical data concerning fluctuating glacier sizes of the past century. Analyzing trends and making conclusion about the makeup of Earth's surface from such data would be an exciting adventure.

**Comment by AaronCJimeny** [comment]
Energy or Climate Geophysics. Working on data regarding geothermal energy or something to do with groundwater monitoring/ contamination
.

**Reply by biavillasboas** [reply]
@AaronCJimeny , see if you can find any problem/dataset in particular that interests you here: https://www.usgs.gov/programs/groundwater-and-streamflow-information-program/groundwater-monitoring

The goal is to start narrowing down the problems you would like to explore by research question, study area, or time period/event of interest

**Reply by AaronCJimeny** [reply]
The USGS has a wide network of wells monitoring water in the High Plains (Ogallala) Aquifer dating back to 2009. These sensors provide measurements of water level and water quality, as well as well log data. A starting research question I would like to pursue would use data from both the stable north end of the aquifer in Nebraska and South Dakota and the rapidly draining southern half in Kansas, Texas, and New Mexico to figure out if the current water level or the rate of water change affects the water quality in the aquifer.

**Comment by seburton** [comment]
I'd like to do a project on landslides. I'm considering focusing on vegetation on hillslopes, or earthquake/extreme weather induced landslides. I've heard of some interesting research from China on these sorts of topics, so perhaps I could find a study site there.

**Reply by seburton** [reply]
I’ve switched gears a bit from research in China after searching for data to use on this project, and now I’m thinking about choosing a study site in California due to the availability of data.
A 2000 paper by Tucker and Bras about Rainfall Variability and landscape evolution piqued my interest when they modeled the potential long-term influence of increased rainfall variability on long term drainage basin morphology; they found that high variability can lead to higher erosion rates, drainage density, and eventually reduced relief. I want to look into the potential impacts of increased rainfall variability on a smaller time scale and relate it to landslide frequency and possibly severity, which could relate to erosion rates. Similar topics to this have been researched in California since Tucker and Bras published that paper, which will be useful for background research, data acquisition, and possible applications. So far I’ve found several papers about rainfall variability in California, as well as a reliable landslide database. Rainfall variability is known to have increased in California during the study period, so hopefully I will see an increase in landslide activity over time. It is possible that an increase in landslide activity can be due to other factors, too, which would be taken into consideration.

**Comment by chloelocke** [comment]
I like space or mineral geophysics. Particularly the sustainable mining aspect of that because I have been learning a lot about that in my mining class and I think it is very interesting.

**Comment by KieranTheYanaway** [comment]
I am personally most intersted in Hazard Geophysics, specifically volcanoes.

**Reply by biavillasboas** [reply]
🌋 This is a great topic, @KieranTheYanaway! Some good starting points would be to check out 
- [this GETSI material](https://serc.carleton.edu/getsi/teaching_materials/volcanic_hazards/unit1.html)
- [the USGS volcano hazards page](https://www.usgs.gov/programs/VHP)

**Reply by KieranTheYanaway** [reply]
To partially solidify my idea for a final project, I am interested in exploring the way that volcanism has changed during the Pleistocene and Holocene. At this website (https://volcano.si.edu/reports_weekly.cfm#vn_342030), there are multiple databases which contain information about volcanic activity during these epochs. I am mainly interested in observing whether any major differences in geologic locations of volcanic eruptions between the Pleistocene and Holocene are apparent from what information these databases (and other sources) provide me. I have not looked into this topic as much as I would like to before completely committing to this project, but I believe this is a fairly solid foundation to spring off of.

**Comment by jackson-krieger** [comment]
Geothermal energy

**Comment by mcorso15** [comment]
Geophys space or geothermal energy

**Comment by biavillasboas** [comment]
@mcorso15 and @jackson-krieger, what aspects of geothermal energy would you be interested in?

- [Here is some inspiration from our neighbors at NREL](https://www.nrel.gov/research/re-geothermal.html)

**Reply by biavillasboas** [reply]
@AaronCJimeny, check this out too since you mentioned geothermal.

**Comment by zmathias16** [comment]
Space Geophysics sounds interesting.

**Comment by pdompier** [comment]
Space geophysics, specifically lunar resources or solar storms, or geothermal energy!

**Comment by biavillasboas** [comment]
Howdy, #spacegeophysics crew! 
Take a moment to explore the resources below to see if you can further develop ideas for your final project 🪐🚀☄️

- [Asteroid catalog](https://3d-asteroids.space/)
- [PDS small bodies node](https://sbn.psi.edu/pds/shape-models/)

@chloelocke @pdompier @anna-nichols @zmathias16 @mcorso15

**Comment by ahamre** [comment]
Oceanography, ocean mining, or something related to the ocean floor. I recently saw an article about Norway finding a large swath of minerals and rare earth metals in the sea bed near the country and am interested in how extracting that would work.

**Comment by madelinepast** [comment]
I would be the most interested in working with planetary or hazards data.
